README.md

```diff
@@ -23,6 +23,10 @@ For details on how to use Velero Plugin for vSphere for each Kubernetes flavor,
 - [vSphere with Kubernetes](docs/supervisor.md)
 - [Tanzu Kubernetes Grid Service](docs/guest.md)
 
+### Backward Compatibility
+
+See documentation [here](docs/backward-compat.md)
+
```

Comment:
Let's add more details in the backward compatibility doc, including the exact Velero versions we have qualified with.